manager/sql/manager_sql.go

```diff
@@ -237,27 +116,41 @@ func (s *SQLManager) create(policy Policy, tx *sqlx.Tx) (err error) {
 		}
 	}
 
-	switch s.db.DriverName() {
-	case "postgres", "pgx":
-		if _, err = tx.Exec(s.db.Rebind("INSERT INTO ladon_policy (id, description, effect, conditions) SELECT ?, ?, ?, ? WHERE NOT EXISTS (SELECT 1 FROM ladon_policy WHERE id = ?)"), policy.GetID(), policy.GetDescription(), policy.GetEffect(), conditions, policy.GetID()); err != nil {
-			return errors.WithStack(err)
-		}
-	case "mysql":
-		if _, err = tx.Exec(s.db.Rebind("INSERT IGNORE INTO ladon_policy (id, description, effect, conditions) VALUES (?, ?, ?, ?)"), policy.GetID(), policy.GetDescription(), policy.GetEffect(), conditions); err != nil {
-			return errors.WithStack(err)
-		}
-	default:
-		return errors.Errorf("Database driver %s is not supported", s.db.DriverName())
+	if _, ok := Databases[s.database]; !ok {
+		return errors.Errorf("Database %s is not supported", s.database)
+	}
+
+	if _, err = tx.Exec(Databases[s.database].QueryInsertPolicy, policy.GetID(), policy.GetDescription(), policy.GetEffect(), conditions); err != nil {
```

Comment:
By using `db.Rebind()` we can actually further streamline the SQL statements and use `?` everywhere instead of dialect-specific parameters.